scripts/globals/trust.lua

```diff
@@ -232,6 +232,11 @@ xi.trust.onTradeCipher = function(player, trade, csid, rovCs, arkAngelCs)
 end
 
 xi.trust.canCast = function(caster, spell, not_allowed_trust_ids)
+    -- GMs can do what they want
+    if caster:getGMLevel() > 0 and caster:checkNameFlags(0x04000000) then
```

Comment:
hey, some of us have minimalist GM's we need to have a flag on but don't give access to a lot of powerful commands on. GM commands give access to trusts if they are enabled. if they are disabled NOBODY server wide should be casting.

Reply:
Agreed 👍